src/test/java/wedlog/address/logic/commands/EditVendorDescriptorTest.java

```diff
@@ -0,0 +1,92 @@
+package wedlog.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static wedlog.address.logic.commands.CommandTestUtil.DESC_BOB;
+import static wedlog.address.logic.commands.CommandTestUtil.DESC_VAL;
+import static wedlog.address.logic.commands.CommandTestUtil.VALID_ADDRESS_BOB;
+import static wedlog.address.logic.commands.CommandTestUtil.VALID_EMAIL_BOB;
+import static wedlog.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static wedlog.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static wedlog.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+
+import org.junit.jupiter.api.Test;
+
+import wedlog.address.logic.commands.VendorEditCommand.EditVendorDescriptor;
+import wedlog.address.testutil.EditVendorDescriptorBuilder;
+
+public class EditVendorDescriptorTest {
+
+    @Test
+    public void equals() {
+        // same values -> returns true
+        EditVendorDescriptor descriptorWithSameValues = new EditVendorDescriptor(DESC_VAL);
+        assertTrue(DESC_VAL.equals(descriptorWithSameValues));
+
+        // same object -> returns true
+        assertTrue(DESC_VAL.equals(DESC_VAL));
+
+        // null -> returns false
+        assertFalse(DESC_VAL.equals(null));
+
+        // different types -> returns false
+        assertFalse(DESC_VAL.equals(5));
+
+        // different values -> returns false
+        assertFalse(DESC_VAL.equals(DESC_BOB));
```

Comment:
Since you have both VAL and VICTOR as vendors, it might be better to use DESC_VICTOR here instead.